HowTos/Controller_Login_SAML_Config.rst

```diff
@@ -37,28 +39,87 @@ If you haven’t already deployed the Aviatrix controller, follow `the Controlle
+#. Click `Add/Update` button
+
+   |image3-1-1|
+
+   |image3-1-2|
+
+   +-------------------------+-------------------------------------------------+
+   | Field                   | Value                                           |
+   +=========================+=================================================+
+   | Endpoint Name           | ``aviatrix_sp_name`` (this is the same name     |
+   |                         | that you chose above)                           |
+   +-------------------------+-------------------------------------------------+
+   | IPD Metadata Type       | Text or URL (depending on what was              |
+   |                         | provided by the SAML provider)                  |
+   |                         | For now, choose URL                             |
+   +-------------------------+-------------------------------------------------+
+   | IDP Metadata Text/URL   | IDP metadata URL/Text copied from the SAML      |
+   |                         | provider configuration                          |
+   |                         | For now, put in a placeholder URL,              |
+   |                         | such as "https://www.google.com"                |
+   +-------------------------+-------------------------------------------------+
+   | Entity ID               | Select `Hostname` for now                       |
+   +-------------------------+-------------------------------------------------+
+   | Access                  | Select admin or read_only access                |
+   +-------------------------+-------------------------------------------------+
+   | Custom SAML Request     | For now leave blank, depending on your specific |
+   | Template                | IDP, you may have to check this option          |
+   +-------------------------+-------------------------------------------------+
+
+#. Click `OK`
+#. Depending on your IDP provider, you may need to upload SP metadata. After temporary SAML endpoint is created:
+
+  - Right click **SP Metadata** button next to the SAML endpoint and save file to your local machine.
+  - Click **SP Metadata** button, and copy the SP metadata as text
 
-This step is usually done by the IDP adminstrator.
+.. _Config_32:
 
-Create a SAML 2.0 app with the following settings that are based on the <aviatrix_sp_name> which is 'controller' for controller login:
+3.2 Create a SAML App for Aviatrix with the IDP
+###############################################
+
+.. note::
+
+   This step is usually done by the IDP administrator.
+   This sections shows only generalized process for creating a SAML application.
+   Refer to the `IDP-specific SAML App Integration <#idp-integration>`_ section for links to detailed steps with each particular IDP.
+
+Create a SAML 2.0 app with the IDP Provider. The following settings are based on the <aviatrix_sp_name>:
 
 #. Assertion Consumer Service URL* = https://aviatrix_controller_hostname/flask/saml/sso/<aviatrix_sp_name>
```

Comment:
Instead of generating from sp_name. Say how it can be retrieved from the controller